src/functions/reportGen.ts

```diff
@@ -39,11 +40,15 @@ const reportGen: Handler = async (event: any, context?: Context, callback?: Call
         await sendATFReport.sendATFReport(generationServiceResponse, visit);
         console.debug("All emails sent, terminating lambda");
```

Comment:
this log shouldn't be here now, this will log on every email! move it out of the loop please